sites/kit.svelte.dev/src/lib/search/SearchBox.svelte

```diff
@@ -91,13 +97,8 @@
 
 	$: if ($searching) {
 		update();
-		document.body.style.position = 'fixed';
 		document.body.style.top = `-${window.scrollY}px`;
-	} else {
-		const scroll = document.body.style.top;
```

Comment:
moved this to `close` and wrapped in check to ensure we were previously searching so that it's not called on page load causing a scroll